src/admin/api.js

```diff
@@ -349,6 +353,102 @@ function extractDestinationCurrency(currencyPair) {
+             * ratePrecision {String} - The number of decimal places to which this currency pair is
+             *      quoted by the partner bank. NOTE: Use this if the base currency is EUR.
+             *      It should be the same as the decimalRate.
+             * invRatePrecision {String} - Used when exchange rates are stored both ways.
+             *      NOTE: Currently not used.
+             * tenor {String} - tenor must be TN
+             * valueDate {String} - The value date for the rate as per the partner bank.
+             *      NOTE: This is currently not mandatory. It can be null or not present.
+             * bidSpotRate {String} The rate where the partner bank buys the base currency at.
+             *      NOTE: Currently, this must be the same as rate but with the decimal point.
+             * offerSpotRate {String} - The rate where the partner bank sells the base currency at.
+             *      NOTE: This is currently not in use.
+             * midPrice {String} - The rate that is at the middle of both the bid and the offer.
+             *      NOTE: This is currently not in use.
+             * validUntilTime {String} - Time and date that this rate will expire.
+             *      Excludes grace period. Format: yyyy-MM-dd HH:mm:ss.SSS in UTC time.
+             *      NOTE: Currently, this must be the same as endTime but in a different format.
+             *      ^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}(\.\d{3})?$
+             * isValid {String} - Indicator that the partner bank was able to provide the rate.
+             *      NOTE: Currently this must always be 'true' or 'TRUE'.
+             * isTradable {String} - A flag to indicate that the currency pair is tradeable.
+             *      NOTE: Currently this must always be 'true' or 'TRUE'.
+             */
+            return {
+                rateSetId: RATE_ID_PER_CURRENCY_PAIR[currencyPair.toUpperCase()],
+                currencyPair: currencyPair.toUpperCase(),
+                baseCurrency: extractSourceCurrency(currencyPair).toUpperCase(),
+                ratePrecision: rateDetails.decimalRate,
+                invRatePrecision: '1', // This is currently not in use.
+                tenor: TENOR_VALUES.TN, // Since payment settlement happens on T+1,
+                // where T = date of order execution, tenor must be TN.
+                valueDate: null, // This is currently not in use.
+                bidSpotRate: buildDecimalRate(rateDetails.rate.toString(), rateDetails.decimalRate),
+                offerSpotRate: '0.0000', // This is currently not in use.
+                midPrice: '0.0000', // This is currently not in use.
+                validUntilTime: rateDetails.endTime.replace('T', ' ').replace('Z', ''),
+                isValid: 'true', // Currently this must always be 'true'
+                isTradable: 'true', // Currently this must always be 'true'
+            };
```

Comment:
```suggestion
            return {
                citi: {
                    rateSetId: RATE_ID_PER_CURRENCY_PAIR[currencyPair.toUpperCase()],
                    currencyPair: currencyPair.toUpperCase(),
                    baseCurrency: extractSourceCurrency(currencyPair).toUpperCase(),
                    ratePrecision: rateDetails.decimalRate,
                    invRatePrecision: '1', // This is currently not in use.
                    tenor: TENOR_VALUES.TN, // Since payment settlement happens on T+1,
                    // where T = date of order execution, tenor must be TN.
                    valueDate: null, // This is currently not in use.
                    bidSpotRate: buildDecimalRate(rateDetails.rate.toString(), rateDetails.decimalRate),
                    offerSpotRate: '0.0000', // This is currently not in use.
                    midPrice: '0.0000', // This is currently not in use.
                    validUntilTime: rateDetails.endTime.replace('T', ' ').replace('Z', ''),
                    isValid: 'true', // Currently this must always be 'true'
                    isTradable: 'true', // Currently this must always be 'true',
                },
            };
```

Reply:
Done.